evap/evaluation/models.py

```diff
@@ -918,6 +918,16 @@ def generate_login_key(self):
     def refresh_login_key(self):
         self.login_key_valid_until = datetime.date.today() + datetime.timedelta(settings.LOGIN_KEY_VALIDITY)
 
+    def get_sorted_contributions(self):
+        return Contribution.objects.filter(contributor=self).order_by('course__semester__created_at', 'course__name_de')
+
+    def get_sorted_courses_participating_in(self):
+        return Course.objects.filter(participants=self).order_by('semester__created_at', 'name_de')
+
+    def get_sorted_courses_voted_for(self):
+        return Course.objects.filter(voters=self).order_by('semester__created_at', 'name_de')
```

Comment:
with the changes proposed in another comment:
`self.contributions, self.courses_participating_in, self.courses_voted_for` instead of `Course.objects.filter(...)`